.tmuxinator.yml

```diff
@@ -26,14 +25,14 @@ windows:
           - fg
         - ln1:
           - sleep 1 # wait for bitcoind
-          - lightningd --dev-fast-gossip --dev-bitcoind-poll=1 --network regtest --bitcoin-rpcuser=bitcoin --bitcoin-rpcpassword=bitcoin --lightning-dir=$FM_LN1_DIR --addr=127.0.0.1:9000 --plugin=$FM_BIN_DIR/ln_gateway --minimint-cfg=$FM_CFG_DIR &
```

Comment:
We should try to keep these flags if possible, maybe only switch them on if supported? Although ideally we could prevent the system `lightningd` leaking into the environment.

Reply:
Added them back in [2e72815](https://github.com/fedimint/minimint/pull/329/commits/2e728153c5ef1e99fc4a9c01c7fa8c074ba4b6e9)